lib/comictaggerlib/comicvinetalker.py

```diff
@@ -233,10 +230,15 @@ def searchForSeries(self, series_name, callback=None, refresh_cache=False):
 
         query_string = urllib.parse.quote_plus(query_string.encode("utf-8"))
 
-        search_url = self.api_base_url + "/search/?api_key=" + self.api_key + "&format=json&resources=volume&query=" + \
-            query_string + \
-            "&field_list=name,id,start_year,publisher,image,description,count_of_issues&limit=100"
-        cv_response = self.getCVContent(search_url + "&page=1")
+        search_url = self.api_base_url + "/search"
+        params = {'api_key': self.api_key,
+                  'format': 'json',
+                  'resources': 'volume',
+                  'query': query_string,
```

Comment:
I am concerned this will double encode because this query_string has already been escaped wtih quote_plus.  I would comment out the query_string = urllib.parse.quotplus(query_string.encode("utf-8")) line and just leave the escaping to requestss params.